forui/lib/src/theme/color_scheme.dart

```diff
@@ -111,6 +115,12 @@ final class FColorScheme with Diagnosticable {
     required this.border,
   });
 
+  /// Creates a disabled color from the given [color].
+  ///
+  /// See:
+  /// * [disabledColorLightness].
+  Color disable(Color color) => HSLColor.fromColor(color).withLightness(disabledColorLightness).toColor();
```

Comment:
Shouldn't we be changing the opacity? On a side note, if we're adding support for disabled, shouldn't this just be a color?

Reply:
I think that changing the opacity isn't ideal, there was a bug with slider where changing the opacity of the active track causes the ticks/marks underneath it to be visible. 

It could be a color but that would mean we need to define all disabled colors for all styles. I'm okay with that but I feel like it should be addressed in a follow-up PR as this PR is chunky enough.

Reply:
Thinking about it again, I think the way forward is to declare a set of disabled colors that all widgets will use, and convert other widgets away from opacity. I got a feeling that messing with the opacity is going to cause some surprising visual oddities down the line...